### DISCUSSION TITLE
Blank / Hung loading a page in CP

### DISCUSSION THREAD

**Discussion by nickcoppins** [statamic/cms#8190] [Troubleshooting]
Hi, 

I'm new to Statamic, looking at migrating our existing long term Laravel site. 

I've setup a test site and installed the Peak starter kit as well as some page builder blocks and a preset. 

I appear to be encountering an issue when having created a test page (which works fine to start with) by adding a block (not sure if it's a specific block at this point or not) and then trying to access it again in the CP, it just loads a blank page / doesn't load the page for editing at all. 

Should I install the debug bar to get more info? Unsure the best way forward to debug what's going on. 

UPDATE: I enabled the debug bar and even it doesn't load on the page in question. 

<img width="1343" alt="image" src="https://github.com/statamic/cms/assets/10858293/14743e99-7645-40ed-a38f-408b5ad2f8cf">

**Comment by duncanmcclean** [comment]
Try running `php artisan statamic:install` - does that help at all?

**Reply by nickcoppins** [reply]
That seemed to install OK, but didn't change anything. 

The page that is loading blank only contains the source / HTML below: 

<img width="607" alt="image" src="https://github.com/statamic/cms/assets/10858293/eba3538a-0cee-45e0-b60a-ab58bd3ce64a">
